src/js/misc.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Event,
+} from '@sentry/types';
+
+/**
+ * Recognizes an Event based on 'exception' property.
+ */
+export function isEvent(event: unknown): event is Event {
+  return typeof event === 'object' && event !== null && 'exception' in event;
+}
```

Comment:
Some other parts of the code do `itemHeader.type == 'event` because an event from `captureMessage` does not have an `exception` but still is an event.

Reply:
Ohh, sorry I missed that. In that case I will make the function `isHardCrash` to accept all the payload items (and solve the existence of exception there) and delete the isEvent.